# You can find control table of Dynamixel on emanual (http://emanual.robotis.com/#control-table)
# Control table item has to be set Camel_Case and not included whitespace
# You are supposed to set at least Dynamixel ID
dyn_tilt:
  ID: 1
  Return_Delay_Time: 0
  CW_Angle_Limit: 0
  CCW_Angle_Limit: 2048
  Moving_Speed: 1